tests/subsys/modem/modem_cmux/src/main.c

```diff
@@ -148,6 +148,14 @@ static uint8_t cmux_frame_dlci2_at_newline[] = {0xF9, 0x0B, 0xEF, 0x05, 0x0D, 0x
 
 static uint8_t cmux_frame_data_dlci2_at_newline[] = {0x0D, 0x0A};
 
+/*************************************************************************************************/
+/*                                     DLCI2 AT CMUX error frames								 */
```

Comment:
The closing `*/` on this line is still way too far. It should be aligned as the rest.

Reply:
I think I'm being fooled by vscode, should be better now. :)